app/views/queues/suggested_edit_complete.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Queues#suggested_edit_complete</h1>
```

Comment:
If these actions always redirect and never render a view, you can delete their view files.

Reply:
I'm not sure if we're going to use this or not. Suggested edits will be a part of #18. So I'll leave the view there, it's not hurting anyone.